nebula-logger/core/main/logger-engine/classes/Logger.cls

```diff
@@ -3048,7 +3048,7 @@ global with sharing class Logger {
 
     // Inner class for saving log entries via the REST API (using the current user's session ID)
     private class RestApiSaver {
-        private final String baseURL = System.Url.getSalesforceBaseUrl().toExternalForm() + '/services/data/v53.0';
+        private final String baseURL = System.Url.getSalesforceBaseUrl().toExternalForm() + '/services/data/v56.0';
```

Comment:
shouldn't this be 57?

Reply:
I think I'm going to bump it in the next release (when I release a new version of the managed package), I still haven't bumped the rest of the codebase to `v57.0` yet. I'm also tinkering with making the code dynamically add the org's current API version to the string so that I don't have to manually update this line of code, but I didn't want to scope-creep this PR too much.

Reply:
Makes sense! And yeah I feel like the good ol' deserialize trick in this one spot would be a good thing.